apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx

```diff
@@ -258,94 +271,139 @@ export const ScriptBasedFlow: FunctionComponent<AdaptiveScriptsPropsInterface> =
-                                                    className="action"
-                                                    data-testid={ `${ testId }-script-template-sidebar-toggle` }
-                                                >
-                                                    <Icon name="bars" />
-                                                </Menu.Item>
-                                            </Menu.Menu>
-                                        )}
-                                    </Menu>
-
-                                    <div className="code-editor-wrapper">
-                                        <CodeEditor
-                                            lint
-                                            language="javascript"
-                                            sourceCode={ sourceCode }
-                                            options={ {
-                                                lineWrapping: true
-                                            } }
-                                            onChange={ (editor, data, value) => {
-                                                setInternalScript(value);
-                                                onScriptChange(value);
-                                            } }
-                                            theme={ isEditorDarkMode ? "dark" : "light" }
-                                            readOnly={ readOnly }
-                                            data-testid={ `${ testId }-code-editor` }
-                                        />
+                                        </div>
                                     </div>
-                                </div>
-                            </Sidebar.Pusher>
-                        </Sidebar.Pushable>
-                    </Grid.Column>
-                </Grid.Row>
-            </Grid>
-        </div>
+                                </Sidebar.Pusher>
+                            </Sidebar.Pushable>
+                        </Grid.Column>
+                    </Grid.Row>
+                </Grid>
+            </div>
+            <ConfirmationModal
+                onClose={ (): void => setShowScriptResetWarning(false) }
+                type="warning"
+                open={ showScriptResetWarning }
+                primaryAction={ t("common:confirm") }
+                secondaryAction={ t("common:cancel") }
+                onSecondaryActionClick={ (): void => setShowScriptResetWarning(false) }
+                onPrimaryActionClick={ (): void => {
+                    setShowScriptResetWarning(false);
+                    resetAdaptiveScriptTemplateToDefaultHandler();
+                } }
+                data-testid={ `${ testId }-delete-confirmation-modal` }
+                closeOnDimmerClick={ false }
+            >
+                <ConfirmationModal.Header data-testid={ `${ testId }-reset-confirmation-modal-header` }>
+                    Are you sure?
```

Comment:
Missing i18n strings
```suggestion
                    Are you sure?
```